Stack/Opc.Ua.Core/Security/Certificates/CertificateValidator.cs

```diff
@@ -385,7 +385,7 @@ public ushort MinimumCertificateKeySize
                     if (m_minimumCertificateKeySize != value)
                     {
                         m_minimumCertificateKeySize = value;
-                        ResetValidatedCertificates();
+                        InternalResetValidatedCertificates();
                     }
```

Comment:
Only when the minimum key size increases the reset must be called.

Reply:
thats about right, but I would still call it for any value changed to follow the same coding pattern.